src/Tribe/Aggregator.php

```diff
@@ -56,149 +56,45 @@ public function __construct() {
 		$this->page        = Tribe__Events__Aggregator__Page::instance();
 		$this->service     = Tribe__Events__Aggregator__Service::instance();
 		$this->pue_checker = new Tribe__PUE__Checker( 'http://tri.be/', 'event-aggregator' );
+		$this->api();
+
+		// THESE ARE JUST HERE FOR TESTING PURPOSES AS WE WAIT FOR THE UI
+		/*
+		$this->api( 'import' )->create( array(
+			'type' => 'manual',
+			'origin' => 'facebook',
+			'source' => '998482400171215',
+			'facebook_app_id' => '',
+			'facebook_secret' => '',
+		) );
+		//*/
+		//$this->api( 'import' )->get( 'd1885e7b2ed7dab8e3d908cecec8780daf55a0ac55e421ed10dadf89f7f51bd1' );
 	}
 
 	/**
-	 * Get event-aggregator origins
-	 *
-	 * @return array
-	 */
-	public function get_origins() {
-		$origins = get_transient( "{$this->cache_group}_origins" );
-
-		if ( ! $origins ) {
-			$origins = $this->service->get_origins();
-
-			if ( is_wp_error( $origins ) ) {
-				return $origins;
-			}
-
-			set_transient( "{$this->cache_group}_origins", $origins, 6 * HOUR_IN_SECONDS );
-		}
-
-		// Let's build out the translated text based on the names that come back from the EA service
-		foreach ( $origins as &$origin ) {
-			$origin->text = __( $origin->name, 'the-events-calendar' );
-		}
-
-		return apply_filters( 'tribe_ea_origins', $origins );
-	}
-
-	/**
-	 * Fetches an image from the service and saves it to the filesystem if needed
+	 * Initializes and provides the API objects
 	 *
-	 * @param string $image_id EA Image ID
+	 * @param string $api Which API to provide
 	 *
-	 * @return stdClass {
-	 *     @type int        $post_id      Attachment ID on WP
-	 *     @type string     $filename     Name of the image file
-	 *     @type string     $path         Absolute path of the image
-	 *     @type string     $extension    Extension of the image
-	 * }
+	 * @return Tribe__Events__Aggregator__API__Origins|Tribe__Events__Aggregator__API__Import|Tribe__Events__Aggregator__API__Image|object
```

Comment:
We should probably add `null` to the list of possible returned types to make it clear they may not receive an API object. Could probably also shorten a little without losing accuracy, ie:

```
@returns Tribe__Events__Aggregator__API__Abstract|null
```